drivers/vmwarevsphere/utils.go

```diff
@@ -35,6 +38,122 @@ func (d *Driver) remoteExec(procman *guest.ProcessManager, arg string) (int64, e
 	return code, nil
 }
 
+func (d *Driver) getDatastore(spec *types.VirtualMachineConfigSpec) (*object.Datastore, error) {
+	var sp *object.StoragePod
+	if d.DatastoreCluster != "" {
+		log.Infof("Finding datastore cluster %s", d.DatastoreCluster)
```

Comment:
do you think we should log in debug? or having it in info makes more sense

Reply:
This bubbles up to the UI and you will see "finding" message first then an error and why it happened. this is just a nicety so you can tell right before the error what it was trying to do